hazelcast-wm/src/main/java/com/hazelcast/web/ClusteredSessionService.java

```diff
@@ -145,6 +151,15 @@ private void reconnectHZInstance() throws ServletException {
         hazelcastInstance = HazelcastInstanceLoader.createInstance(this, filterConfig, properties);
         clusterMap = hazelcastInstance.getMap(clusterMapName);
         sss = (SerializationServiceSupport) hazelcastInstance;
+        try {
+            Config hzConfig = hazelcastInstance.getConfig();
+            MapConfig mapConfig = hzConfig.getMapConfig(clusterMapName);
+            mapConfig.setMaxIdleSeconds(Integer.parseInt(sessionTTL));
+            hzConfig.addMapConfig(mapConfig);
+
+        } catch (UnsupportedOperationException ignored) {
+            LOGGER.info("Client cannot access config.");
```

Comment:
I would suggest to create a fail fast check so if `session-ttl-seconds` are set and mode is client/server, we should throw  InvalidConfigException.